tox.ini

```diff
@@ -1,47 +1,53 @@
 [tox]
 envlist = py27,pypy,py33,py34,py35,flake8
+deps =
+    pytest
+    unicodecsv
+    hypothesis>=3.6.1,<4.0.0
```

Comment:
Tox lets you specify dependencies once and then refer to them as a group, so I've refactored accordingly.